packages/terra-clinical-header/src/Header.jsx

```diff
@@ -23,7 +23,7 @@ const propTypes = {
   /**
    * Text to be displayed as the title in the header bar
    */
-  title: PropTypes.string,
+  text: PropTypes.string,
```

Comment:
Can this and the other new props be alphabetized for consistency?

Reply:
Arranged [53ced76](https://github.com/cerner/terra-clinical/pull/792/commits/53ced7677e4fd6eb2b3bea9465b779d456355f14)